tests/phpunit/TestFramework/PhpUnit/Adapter/PestAdapterTest.php

```diff
@@ -275,6 +287,21 @@ public function outputProvider(): iterable
         yield ['Tests:  1 failed, 8 passed', false];
     }
 
+    public function syntaxErrorOutputProvider(): iterable
+    {
+        yield ['Tests:  1 risked', false];
+
+        yield [
+            <<<TXT
+'ParseError
```

Comment:
Just in case: you can now indent the heredocs: https://wiki.php.net/rfc/flexible_heredoc_nowdoc_syntaxes

I would also recommend to use nowdocs vs heredocs when possible (i.e. quote `TXT` into `'TXT'`)

Reply:
fixed in https://github.com/infection/infection/pull/1557